Userland/Applications/FileManager/FileOperationProgressWidget.cpp

```diff
@@ -55,6 +55,9 @@ FileOperationProgressWidget::FileOperationProgressWidget(NonnullRefPtr<Core::Fil
             did_error();
             return;
         }
+
+        dbgln(line);
```

Comment:
The first argument to `dbgln` should be a format string:
```suggestion
        dbgln("{}", line);
```

Reply:
I just forgot that there from debugging and removed it entirely.